### PR TITLE
[ISSUE #291]Add Translation for "Dledger" Section in the v4.x & 5.0 Document (CN -> EN)

### LINKED ISSUE #291: Translation for "Best Practices-Dledger" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "Best Practices-Dledger" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #291 

## Brief changelog

- Add Translation for "Dledger" Section in the v4.x & 5.0 Document (CN -> EN)

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md [hunk 1]
```diff
@@ -1,83 +1,91 @@
...
-DLedger是一套基于Raft协议的分布式日志存储组件，部署 RocketMQ 时可以根据需要选择使用DLeger来替换原生的副本存储机制。本文档主要介绍如何快速构建和部署基于 DLedger 的可以自动容灾切换的 RocketMQ 集群。
+# DLedger
 
-### 1. 源码构建
-构建分为两个部分，需要先构建 DLedger，然后 构建 RocketMQ。
+## DLedger Quick Deployment
 
-#### 1.1 构建 DLedger
+### Preface
+DLedger is a set of distributed log storage components based on Raft protocol. When deploying RocketMQ, you can choose to use DLeger to replace the native replica storage mechanism. This document is mainly introduced for how to build and deploy auto failover RocketMQ cluster based on DLedger.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: DLeger -> DLedger

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md [hunk 2]
```diff
@@ -1,83 +1,91 @@
...
+
+### 1. Build from source code
+Build phase contains two parts, first, build DLedger, then build RocketMQ.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: The build phase is divided into two parts, DLedger should be built first, and then build RocketMQ.

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md [hunk 3]
```diff
@@ -1,83 +1,91 @@
...
+$ mvn -Prelease-all -DskipTests clean install -U
 ```
-### 2. 快速部署
 
-在构建成功后
-```shell
-cd distribution/target/apache-rocketmq
+### 2. Quick Deployment
+
+after build successful
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: After building successfully,

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md [hunk 4]
```diff
@@ -1,83 +1,91 @@
...
 
-sh bin/dledger/fast-try.sh start
+```shell
+#{rocketmq-version} replace with rocketmq actual version. example: 5.0.0-SNAPSHOT
+$ cd distribution/target/rocketmq-{rocketmq-version}/rocketmq-{rocketmq-version}
+$ sh bin/dledger/fast-try.sh start
 ```
 
-如果上面的步骤执行成功，可以通过 mqadmin 运维命令查看集群状态。
+if the above commands executed successfully, then check cluster status by using mqadmin operation commands.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: if -> If

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md [hunk 5]
```diff
@@ -1,83 +1,91 @@
...
 
 ![ClusterList](https://img.alicdn.com/5476e8b07b923/TB11Z.ZyCzqK1RjSZFLXXcn2XXa)
 
-（BID 为 0 的表示 Master，其余都是 Follower）
+（BID is 0 indicate Master, the others are Follower）
+
+After startup successful, producer can produce message, and then test failover scenario.
 
-启动成功，现在可以向集群收发消息，并进行容灾切换测试了。
+Stop cluster fastly, execute the following command:
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: Execute the following command to stop the cluster quickly:

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md [hunk 6]
```diff
@@ -1,83 +1,91 @@
...
-顺利的话，会看到如下内容：
+
+If everything goes well, the following content will appear:
 
 ![ClusterList](https://img.alicdn.com/5476e8b07b923/TB11Z.ZyCzqK1RjSZFLXXcn2XXa)
 
-（BID 为 0 的表示 Master，其余都是 Follower）
+（BID is 0 indicate Master, the others are Follower）
+
+After startup successful, producer can produce message, and then test failover scenario.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: After the startup is successful

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md [hunk 7]
```diff
@@ -1,83 +1,91 @@
...
-快速部署，默认配置在 conf/dledger 里面，默认的存储路径在 /tmp/rmqstore。
 
+Quick deployment, default configuration is in directory conf/dledger, default storage path is /tmp/rmqstore.
 
-### 3. 容灾切换
 
-部署成功，杀掉 Leader 之后（在上面的例子中，杀掉端口 30931 所在的进程），等待约 10s 左右，用 clusterList 命令查看集群，就会发现 Leader 切换到另一个节点了。
+### 3. Failover
 
+After successful deployment, kill Leader process(as the above example, kill process that binds port 30931), about 10 seconds elapses, use clusterList command check cluster's status, Leader switch to another node.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: After the successful deployment, kill the Leader process(as the above example, kill process that binds port 30931), then wait for 10 seconds, use clusterList command to check cluster status, and you could find that the Leader has been switched to another node.

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Hi @mxsm ,

Thx for your contribution to the translation task. Please make some changes to the word usage and sentences.

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md (hunk 7 above)
Comment (mxsm) [reply i18n/en/docusaurus-plugin-content-docs/version-5.0/06-最佳实践/16dledger.md]: @tsunghanjacktsai  I will modify and resubmit this PR

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**